### PR TITLE
Healpy

### PR DESCRIPTION
**Pull request by AlecThomson**
Adding the Healpy python library

### CODE REVIEW

**Comment by AlecThomson** [comment]
Hey @astrofrog, sorry I managed to bungle the formatting with this PR and #85. I believe that should be sorted now, and should hopefully pass the workflow check

File: entries/codes/healpy.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: healpy
+category: codes
+tags:
+text: Some of the results in this paper have been derived using the healpy and HEALPix packages
```
Comment (hamogu) [inline entries/codes/healpy.yaml]:
```suggestion
text: Some of the results in this paper have been derived using the healpy and HEALPix packages.
```

File: entries/codes/healpy.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  keywords = {Cosmology: Cosmic Microwave Background, Cosmology: Observations, Methods: Statistical},
+      year = 2005,
+      month = apr,
+    volume = 622,
+      pages = {759-771},
+        doi = {10.1086/427976},
+    adsurl = {http://adsabs.harvard.edu/abs/2005ApJ...622..759G},
+    adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+  }
+notes: At the first use of the HEALPix acronym, a footnote placed in the main body of the paper referring to the HEALPix web site, currently http://healpix.sf.net
```
Comment (hamogu) [inline entries/codes/healpy.yaml]:
```suggestion
notes: At the first use of the HEALPix acronym, a footnote should be placed in the main body of the paper referring to the HEALPix web site, currently http://healpix.sf.net .
```

**Review by hamogu — COMMENTED** [review]
I see two trivial typos (period missing at end of sentence). I'll fix that myself and then merge when CI passes. Thanks a lot!

**Pull request merged**